music21/stream/base.py

```diff
@@ -2908,6 +2908,14 @@ def splitAtDurations(self, *, recurse=False) -> base._SplitTuple:
         >>> result = m.splitAtDurations(recurse=True)
         >>> list(result[0][note.Rest])
         [<music21.note.Rest whole>, <music21.note.Rest quarter>]
+
+        >>> r2 = note.Rest(quarterLength=5.0)
+        >>> r2.fullMeasure = True
+        >>> m2 = stream.Measure(r2)
+        >>> m2.insert(0, meter.TimeSignature('6/4'))
```

Comment:
I think that `5/4` would make this clearer, no?

Reply:
Needs a comment to explain, clearly. This is showing that when the rest doesn't fill the measure but we set `.fullMeasure` = True anyhow, we still treat as full measure and don't split it.